src/test/regress/expected/multi_router_planner.out

```diff
@@ -2308,6 +2310,7 @@ DETAIL:  distribution column value: 1
 INSERT INTO articles_hash VALUES (51,  1, 'amateus', 1814);
 DEBUG:  Creating router plan
 DEBUG:  Plan is router executable
+DETAIL:  distribution column value: 1
```

Comment:
might want to add a multi-row insert with the same partition column value in each row to get code coverage for that

Reply:
Added, but it does not get printed here. We set the partition column on the job only after we realize that multi-column insert is a router executable query.